### PR TITLE
adding new vulnerability info - CVE-2022-23307 

### PR DESCRIPTION
**Pull request by scott-confluent**
See google doc for more info: https://docs.google.com/document/d/16nWAWysgPPnXaLb2FdsZIo1fe6qwmh6Q4PagxYc4ps8/edit#heading=h.77psxmd0wrk

more details: https://nvd.nist.gov/vuln/detail/CVE-2022-23307

### CODE REVIEW

File: cve-list.html [hunk 1]
```diff
@@ -9,6 +9,31 @@ <h1>Apache Kafka Security Vulnerabilities</h1>
...
+  
+  <table class="data-table">
+  <tbody>
+    <tr>
+      <td>Versions affected</td>
+      <td>All AK versions</td>
+    </tr>
+    <tr>
+      <td>Fixed versions</td>
+      <td>NA</td>
```
Comment (junrao) [inline cve-list.html]: Could we change NA to "In the absence of a new log4j 1.x release, one can remove Chainsaw from the log4j-1.2.17.jar artifact."?

**Review by junrao — COMMENTED** [review]
@scott-confluent : Thanks for the PR. Left one comment below.

File: cve-list.html (hunk 1 above)
Comment (rite2nikhil) [reply cve-list.html]: makes sense

Comment (scott-confluent) [reply cve-list.html]: Updated.

**Review by junrao — APPROVED** [review]
@scott-confluent : Thanks for the updated PR. LGTM

**Pull request merged**